client/account-recovery/controller.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import LostPasswordPage from 'account-recovery/lost-password';
+import { getCurrentUser } from 'state/current-user/selectors';
+
+export function lostPassword( context, next ) {
+	context.primary = <LostPasswordPage basePath={ context.path } className="account-recovery__container" />;
```

Comment:
I switched it `is-section-account-recovery .main` in https://github.com/Automattic/wp-calypso/pull/8955/commits/834730a3a40cba5297d1ab8af00aba81dc54af3a